content/swiftui/concepts/viewmodifier/terms/fontWeight/fontWeight.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: '.fontWeight()'
+Description: 'Sets the font weight of the text in Text view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.fontWeight()`** modifier sets the font weight. We can apply the following weights:
+
+- ultraLight
+- thin
+- light
+- regular
+- medium
+- semibold
+- bold
+- heavy
+- black
+
+## Syntax
+
+```pseudo
+Text("your ultraLight text")
+  .fontWeight(.ultraLight)
+```
+
+## Example
+
+In the following example we use `regular` and `bold` expressions to set font weight.
+
+```swift
+import SwiftUI
+
+struct SomeView: View {
+  var body: some View {
+    VStack {
+      Text("new font Weight")
```

Comment:
Rather than both saying "new Font Weight" can we have them say "Regular Font Weight" and "Bold Font Weight"?